src/pages/[platform]/build-a-backend/restapi/cancel-api-requests/index.mdx

```diff
@@ -0,0 +1,41 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Cancel API requests',
+  description:
+    "Cancelling in-flight API calls in Amplify",
+  platforms: [
+    'javascript',
+    'react-native',
+    'angular',
+    'nextjs',
+    'react',
+    'vue'
+  ]
+};
+
+export const getStaticPaths = async () => {
+  return getCustomStaticPath(meta.platforms);
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+import js0 from '/src/fragments/lib/restapi/js/cancel.mdx';
+
+<Fragments
+  fragments={{
+    javascript: js0,
+    angular: js0,
+    nextjs: js0,
+    react: js0,
+    vue: js0,
+    'react-native': js0,
+  }}
+/>
```

Comment:
is this fragment reused from gen1? in the new docs we're trying to move away from fragments as they're tough to track changes to across pages. Can the content be ported in?